deb/openmediavault/var/www/openmediavault/css/theme-all.scss

```diff
@@ -622,3 +622,19 @@ div#headerrlogo {
 .x-webkit :not(.x-form-textarea-body) > .x-form-trigger-wrap {
 	height: initial;
 }
+/******************************************************************************/
+/* Fix dashboard widgets multi-cols alighment */
+.x-dashboard-column-first {
```

Comment:
Can you please explain why this is necessary?

Reply:
It's an attempt to align the widgets with each other.
Before the addition:
![Screenshot from 2020-02-05 22-03-46](https://user-images.githubusercontent.com/53077134/73874215-22714700-4864-11ea-8e5e-314b8b481a0b.png)

After the addition:
![Screenshot from 2020-02-05 22-04-01](https://user-images.githubusercontent.com/53077134/73874226-256c3780-4864-11ea-9063-d8718e1bea9b.png)

Alsp

Reply:
I can not reproduce this. I think we should open a separate PR for that.